ansible_collections/arista/avd/roles/eos_designs/docs/how-to/avd-to-cvaas.md

```diff
@@ -0,0 +1,134 @@
+# The purpose of this tutorial is to review the steps required in order to integrate AVD with Arista CVaaS.
+
+![Figure: 1](../../../../media/create_account.png)
+
+## Within CVaaS navigate to:
+### Settings and Tools --> Access Control --> Service Accounts --> "New Service Account"
```

Comment:
new line between headings

Reply:
might be nice to also has more screenshots, feel free to re-use the screenshots from our ansible-cvp documentation: https://github.com/aristanetworks/ansible-cvp/blob/devel/ansible_collections/arista/cvp/docs/how-to/cvp-authentication.md#how-to-generate-service-account-tokens

Reply:
Yep, I like your screenshots better so I updated to them.
Added the line between headings.